prettyurl.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# For testing:
+# rm -rf nextcloud-snap
+# git clone --branch prettyurl https://github.com/nextcloud/nextcloud-snap.git
+# cd nextcloud-snap
+
+# Get the correct tar file from the branch itself
+NC_DOMAIN="$(grep 'https://download.nextcloud.com/server' ./snap/snapcraft.yaml | grep -oP 'https://.*')"
+
+# Download the tar file
+curl -fL "$NC_DOMAIN" -o /tmp/ncpackage.tar.bz2
+
+# Extract the archive
```

Comment:
I think the sha256 hash of 'ncpackage.tar.bz2' should be checked against the one listed in snapcraft.yml file here.

Reply:
I don't think that this is necessary because it only uploads any change to the .htaccess file jn the curent origin developing branch and not any production branch. But I'll let @kyrofa decide!